src/ontology/mondo-edit.obo

```diff
@@ -376435,7 +376399,10 @@ xref: ICD10CM:Q39.6 {source="Orphanet:91358/attributed", source="Orphanet:91358/
 xref: ICD9:750.4 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: Orphanet:91358 {source="MONDO:equivalentTo"}
 xref: SCTID:204667006 {source="MONDO:equivalentTo"}
-is_a: MONDO:0015207 {source="Orphanet:91358"} ! non-syndromic esophageal malformation
+is_a: MONDO:0020019 ! digestive tract malformation
+is_a: MONDO:0021147 ! disorder of development or morphogenesis
+relationship: disease_has_location UBERON:0001043 ! esophagus
```

Comment:
did you add this manually?

Reply:
these were added in the rewiring (hence they don't have a source) and I think I missed these (otherwise I would have added a source). We should double check these - if they should be kept or deleted (I'll do it after you are done with your review, I don't want to make changes and then this diff + comments will get all wonky)